### PR TITLE
RD-3377 Support Standing Order

### PR DESCRIPTION
**Pull request by yuhanniu**

### CODE REVIEW

File: app/src/main/resources/index.html [hunk 1]
```diff
@@ -6,12 +6,19 @@
...
+        <label for="buttonTypePopup">Popup</label>
+        <input type="radio" id="buttonTypePopup" name="buttonTypeSelector" value="POPUP" />
+        <small>* Mobile is always redirect</small>
+    </div>
+    <div class="transferTypeSelector">
+        Select transfer type:
+        <label for="transferTypeSingleImmediate">Single Immediate Transfer (default)</label>
+        <input type="radio" id="transferTypeSingleImmediate" name="transferTypeSelector" value="SINGLE_IMMEDIATE" checked />
+        <label for="transferTypeStandingOrder">Standing Order</label>
+        <input type="radio" id="transfertypeStandingOrder" name="transferTypeSelector" value="STANDING_ORDER" />
```
Comment (grekinsky) [inline app/src/main/resources/index.html]: id should be `transferTypeStandingOrder`, type with capital 'T'

File: app/src/main/resources/index.html [hunk 2]
```diff
@@ -6,12 +6,19 @@
...
-      <input type="radio" id="buttonTypePopup" name="buttonTypeSelector" value="POPUP" />
-      <small>* Mobile is always redirect</small>
+        Desktop mode:
+        <label for="buttonTypeRedirect">Redirect (default)</label>
+        <input type="radio" id="buttonTypeRedirect" name="buttonTypeSelector" value="REDIRECT" checked />
+        <label for="buttonTypePopup">Popup</label>
+        <input type="radio" id="buttonTypePopup" name="buttonTypeSelector" value="POPUP" />
+        <small>* Mobile is always redirect</small>
+    </div>
+    <div class="transferTypeSelector">
```
Comment (grekinsky) [inline app/src/main/resources/index.html]: maybe add some margin at the bottom of `.transferTypeSelector` in stylesheet to separate it from the button

Comment (yuhanniu) [reply app/src/main/resources/index.html]: Actually I was wondering about that, how would you do that? Should I add a new line or something like that?

Comment (grekinsky) [reply app/src/main/resources/index.html]: Yes it's similar to the styles from `. buttonTypeSelector`:
```
. transferTypeSelector {
    display: block;
    margin-bottom: 20px;
}

.transferTypeSelector label {
    cursor: pointer;
    margin-left: 10px;
}
```

Comment (yuhanniu) [reply app/src/main/resources/index.html]: thank you, CSS has been mysterious to me :)

**Pull request merged**